app/src/main/res/values-de/strings-de.xml

```diff
@@ -485,8 +485,306 @@
+	<string name="g5_debug_settings">G5 Debugeinstellungen</string>
+	<string name="g5_experimental_mode">Experimenteller Modus, der keine Rohdaten benutzt</string>
+	<string name="g5_force_ui_thread">Dies ist ausschlaggebend bei einigen Android-Geräten zum korrekten Verbinden, kann aber auch verpasste Werte verursachen, wenn andere UI intensive Activities benutzt werden.</string>
+	<string name="g5_full_authentification">Dies wird eine volle Authentifikation bei jedem Leseversuch versuchen. Notwending für neuere G5-Transmitter Firmwareversionen.</string>
+	<string name="g5_remove_before_read">Dies wird das G5 von der Paring-Geräte-Liste deines Geräts entfernen und eine volle Authentifikation und Bonding bei jedem Lesen versuchen.</string>
+	<string name="g5_scan_constantly">Einige Geräte funktionieren besser, wenn Sie kostant scannen, andere nicht. Wenn das Lesen zuverlässig ist, wenn diese Einstellung nicht ausgewählt ist, solltest du eine längere Batterielaufzeit haben. Normalerweise ist es am am besten dies deaktiviert zu lassen.</string>
+	<string name="glucose_meter_options">Optionen für kabellose Blutzuckermessgeräte</string>
+	<string name="glucose_meters">Blutzuckermessgräte</string>
+	<string name="hours_between_calibrations">Stunden zwischen den Kalibrierungen</string>
+	<string name="how_receive_data">Wie wirst du Daten von deinem Dexcom/Transmitter empfangen?</string>
+	<string name="import_db_warning">\n        WARNUNG: Das Importieren eine Datenbank kann einen Datenverlust verursachen. Falls Sie nicht wissen, warum Sie das machen sollten, benutzen Sie es nicht.\n        Im Normalbetrieb von xDrip wird dies nicht benötigt. Benutzen Sie es nur im Falle eines vorherigen Datenverlustes.\n        Falls Sie nicht sicher sind, was zu tun ist, Sie aber dennoch den Import benutzen wollen, fragen Sie einen erfahrenen Benutzer.\n\n\n        Anleitung:\n\n        Platzieren Sie die Datenbank, die Sie installieren wollen, im Ordner \\\'xdrip\\\' auf dem primären Speicher\n        (manchmal auch \\\'interner Speicher\\\' vom Dateiexplorer genannt)\n        Das ist derselbe Ordner in den xDrip auch exportiert.\n        \n        Falls Sie eine eine ZIP-Datei haben, entpacken Sie sie (normalerweise lang drücken->entpacken)\n        Falls sie in einem Unterordner entpackt werden sollte, wird sie immernoch gefunden werden.\n \n        Nachdem die Datei platziert ist, starten Sie die Import-Routine.\n        Scrollen Sie zum korrekten Datenbanknamen, klicken Sie darauf und folgen Sie den Anweisungen auf dem Bildschirm.\n \n \n        WARNUNG: Eine Datenbank kann nur importiert werden, wenn sie dieselbe Version hat, die momentan von xDrip benötigt wird.\n        Die Datenbankversionen werden sich mit zukünftigen xDrip Aktualisierungen vielleicht ändern.\n        Es ist am besten, wenn Sie die Datenbank mit derselben Version von xDrip importieren, mit der sie auch exportiert wurde, und xDrip dann aktualisieren.</string>
```

Comment:
I can't find this string (`import_db_warning`) in the general strings.xml of this PR.